File: VB/DXSample/Data/ViewModel.vb
Imports System
Imports System.Collections.ObjectModel

Namespace DXSample

    Public Class ViewModel

        Public Property Collection As ObservableCollection(Of SimpleData)

        Public Sub New()
            Dim rand As Random = New Random()
            Collection = New ObservableCollection(Of SimpleData)()
            For i As Integer = 0 To 100 - 1
                Collection.Add(New SimpleData() With {.ID = i, .Name = "Name" & i, .Description = "Description" & i, .Price = rand.Next(0, 10000) \ rand.Next(1, 10), .Count = rand.Next(0, 1000)})
            Next
        End Sub
    End Class
End Namespace

File: CS/DXSample/Data/ViewModel.cs
using System;
using System.Linq;
using System.Collections.Generic;
using System.Collections.ObjectModel;

namespace DXSample {
    public class ViewModel {

        public ObservableCollection<SimpleData> Collection { get; set; }
        
        public ViewModel() {
            Random rand = new Random();
            Collection = new ObservableCollection<SimpleData>();
            for (int i = 0; i < 100; i++) {
                Collection.Add(new SimpleData() { 
                    ID = i,
                    Name = "Name"+i,
                    Description = "Description" + i,
                    Price = rand.Next(0, 10000) / rand.Next(1, 10),
                    Count = rand.Next(0, 1000)
                });
            }
        }
    }
}